src/components/ReviewCard.jsx

```diff
@@ -0,0 +1,16 @@
+export default function ReviewCard ({owner,review_body, category, title, review_img_url, review_id, votes}) {
+    return(
+        <>
+        <br />
+        <section id="ReviewCard">
+            <h3><strong>{title}</strong></h3>
+            {/* <p>{review_body}</p> */}
+            <img src={review_img_url} alt ={review_id} className="ReviewImg"/>
```

Comment:
It might be nice to be more descriptive in the alt tag so anyone using a screen reader gets an idea of what the image is for.